components/burger/Burger.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from "../burger/Burger.module.css";
+
+// export type BurgerProps = {
+//   primary: boolean;
+// };
+
+const Burger = () => {
+  return (
+    <div className={`${styles.burger}`}>
```

Comment:
```suggestion
    <div className={styles.burger}>
```
I saw that silvia did the same. doesn't it work without backticks?!?

Reply:
danke :-)